### PR TITLE
Bug 1992592: Atomic List Type

### PR DESCRIPTION
**Pull request by EmilyM1**
Making list atomic to prevent GitOps overwrite

### CODE REVIEW

File: config/v1/types_oauth.go [hunk 1]
```diff
@@ -31,6 +31,7 @@ type OAuthSpec struct {
 	// identityProviders is an ordered list of ways for a user to identify themselves.
 	// When this list is empty, no identities are provisioned for users.
 	// +optional
+	// listType=atomic
```
Comment (soltysh) [inline config/v1/types_oauth.go:34]: It's a tag, so it should be `// +listType=atomic`

Comment (EmilyM1) [reply config/v1/types_oauth.go]: Ty!

**Review by soltysh — APPROVED** [review]
/lgtm
/approve

File: config/v1/types_oauth.go (hunk 1 above)
Comment (sttts) [reply config/v1/types_oauth.go]: I totally miss a discussion why atomic is correct here. I doubt it is.

Comment (sttts) [reply config/v1/types_oauth.go]: "Making list atomic to prevent GitOps overwrite" – atomic enforces exactly that: overwrite.

**Pull request merged**